cypress/integration/ContributionForm_spec.js

```diff
@@ -13,74 +13,101 @@ describe('ContributionForm View', () => {
+
     cy.get('form input[name="city"]').type('Paris')
       .get('form input[name="city"]').should('have.value', 'Paris')
   });
+
   it('Should have a controlled input field for state whose value reflects the data typed into the form', () => {
+
     cy.get('form select[name="state"]').select('Kansas')
       .get('form select[name="state"]').should('have.value', 'Kansas')
   });
+
   it('Should have a controlled input field for zip code whose value reflects the data typed into the form', () => {
+
     cy.get('form input[name="zip"]').type('00000')
       .get('form input[name="zip"]').should('have.value', '00000')
   });
+
   it('Should have a controlled input field for phone number whose value reflects the data typed into the form', () => {
+
     cy.get('form input[name="phone"]').type('1-800-GOOD-DOC')
       .get('form input[name="phone"]').should('have.value', '1-800-GOOD-DOC')
   });
 
   it('should display the sumbit page when the sumbit button is pressed', () => {
+
     cy.get('#submit-button').click()
       .location('pathname').should('eq', '/submit')
       .get('h2').should('contain', 'Contribution Form')
       .get('form input[name="firstName"]')
   });
 
-  // describe('ContributionForm Error Handling', () => {
-  //   beforeEach(() => {
-  //     cy.get('form input[name="firstName"]').type('Betty')
-  //       .get('form input[name="lastName"]').type('White')
-  //       .get('form input[name="specialty"]').type('trans health')
-  //       .get('form input[name="insurance"]').type('sliding scale')
-  //       .get('form input[name="address"]').type('123 Main St')
-  //   })
-  //   it('Should return an error message if the user does not fill out all input fields',() => {
-  //     cy.get('form > button').click()
-  //       .get('.error-msg').should('have.text', 'Error: Please fill out all input fields')
-  //   })
-  //   it('Should display a message stating that the suggestion was successfully receieved', () => {
-  //     .intercept('POST', 'backendAPI', {
-  //       statusCode: 200
-  //     })
-  //     .get('form > button').click()
-  //     .get('.confirm-msg').should('have.text', 'Thank you for your suggestion! We will review your submission for approval.')
-  //   })
-  // })
+  describe('ContributionForm Error Handling', () => {
+    beforeEach(() => {
+
+      cy.fixture('mockPost.json')
+        .then(mockPost => {
+          cy.intercept('POST', 'https://head-to-toe-be.herokuapp.com/api/v1/medical_professionals', {
+            statusCode: 200,
+            delay: 100,
+            body: mockPost
+          })
+        })
+      cy.visit('http://localhost:3000')
+
+      .get('form input[name="firstName"]').type('Betty')
+      .get('form input[name="lastName"]').type('White')
+      .get('form input[name="specialty"]').type('trans health')
+      .get('form input[name="insurance"]').type('sliding scale')
+      .get('form input[name="address"]').type('123 Main St')
```

Comment:
Deleted for ease of transfer of information.  I think we should add the individual lines for the test to test for full user flow though.  

I pulled from main after your last merge.  However, the test remained.  Since its only testing for any "string" it still passes in cypress.  We can make this more accurate though, in our next refactor branch I think.